src/ui/Paginator/Paginator.tsx

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
```

Comment:
Please move this to `pages/AdminPage`, or remove it and use `components/Paginator` instead

Reply:
The reason I created this one is to add support for ellipses for mobile version.